keps/sig-node/2040-cri-beta/README.md

```diff
@@ -0,0 +1,264 @@
+
+# Promote CRI API to beta
```

Comment:
let's just use this KEP to identify changes from the alpha to beta and next release beta to stable.

So let's title this: "Kubelet CRI support", so it matches the tracking issue.
https://github.com/kubernetes/enhancements/issues/2040